Xamarin.Forms.Core/Items/ListItemsLayout.cs

```diff
@@ -4,7 +4,7 @@ namespace Xamarin.Forms
 {
 	public class ListItemsLayout : ItemsLayout
 	{
-		public ListItemsLayout(ItemsLayoutOrientation orientation) : base(orientation)
+		public ListItemsLayout([Parameter("Orientation")] ItemsLayoutOrientation orientation) : base(orientation)
```

Comment:
wasn't this already added in an other PR ?

Reply:
Yes, but that PR was to master. And I needed the change here for one of the test pages. (Most of the code in this PR was written months ago, it just never quite got finished.)